### PR TITLE
Idempotency and CreatePool edits

### PR DESCRIPTION
**Pull request by mulkieran**
Related: https://github.com/stratis-storage/project/issues/51.

### CODE REVIEW

File: docs/dbus/DBusAPIReference.lyx [hunk 1]
```diff
@@ -484,47 +524,72 @@ Methods
...
 \begin_layout Description
-Returns: a list of object paths of the blockdevs added to the pool.
- Signature: a(o)qs.
+Returns: Signature: (bao)qs.
 \end_layout
 
+\begin_deeper
+\begin_layout Enumerate
+True if any cache devices were added, otherwise false.
+ Signature: s.
```
Comment (jbaublitz) [inline docs/dbus/DBusAPIReference.lyx]: I think this should this be `b` here.

**Comment by mulkieran** [comment]
Whoops!

**Comment by mulkieran** [comment]
Since we plan to cut a release quite soon, it seems reasonable to merge this.

**Comment by mulkieran** [comment]
rebased.

**Pull request merged**